### PR TITLE
Fix existed and added missed unit tests.

### PR DESCRIPTION
**Pull request by michal1990**

### CODE REVIEW

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/TagsManager.java [hunk 1]
```diff
@@ -47,19 +47,19 @@ boolean editTag(Tag tag) {
     }
 
     boolean clearTag(Tag tag) {
-        /*if (tag.getKey().equals(TAG_GPS_TIMESTAMP)) {
+        if (tag.getKey().equals(TAG_GPS_TIMESTAMP)) {
```
Comment (kejmil01) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/TagsManager.java]: There is a Timestamp inputType now.

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/TagsManager.java [hunk 2]
```diff
@@ -47,19 +47,19 @@ boolean editTag(Tag tag) {
...
             tag.setValue("00:00:00");
         } else {
-            switch (tag.getObjectType()) {
-                case STRING:
+            switch (tag.getTagType().getInputType()) {
+                case TEXT_STRING:
```
Comment (kejmil01) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/TagsManager.java]: You have to cover all types.

File: app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/EditViewModelTest.java [hunk 3]
```diff
@@ -1,22 +1,37 @@
...
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(RobolectricTestRunner.class)
 @Config(constants = BuildConfig.class, sdk = 21)
 public class EditViewModelTest {
 
-    /*private static final Tag TAG = new Tag("key", "value", InputType.STRING);
+    private static final Tag TAG = new Tag("key", "value", TagType.build(InputType.DATE_STRING));
```
Comment (kejmil01) [inline app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/EditViewModelTest.java]: Do you really need this field to be static?

File: app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/adapter/EditCompoundRecyclerViewTest.java [hunk 4]
```diff
@@ -1,15 +1,29 @@
...
-//@RunWith(RobolectricTestRunner.class)
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+@RunWith(RobolectricTestRunner.class)
 @Config(constants = BuildConfig.class, sdk = 21)
 public class EditCompoundRecyclerViewTest {
 
-    /*private static final Tag TAG = new Tag("key", "value", InputType.STRING);
+    private static final Tag TAG = new Tag("key", "value", TagType.build(InputType.TEXT_STRING));
```
Comment (kejmil01) [inline app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/adapter/EditCompoundRecyclerViewTest.java]: Do you need this field to be static?

File: app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/adapter/EditItemAdapterTest.java [hunk 5]
```diff
@@ -56,10 +77,10 @@ public void whenTagIsUpdated_TagsListIsUpdated() {
         adapter.updateImageList(TAGS_LIST);
 
         //When
-        Tag editedTag = new Tag("key", "editedValue", InputType.STRING);
+        Tag editedTag = new Tag("key", "editedValue", TagType.build(InputType.TEXT_STRING));
         adapter.updateTag(editedTag);
 
         //Then
         assertThat(adapter.getTagsList(), containsTag(editedTag));
```
Comment (kejmil01) [inline app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/adapter/EditItemAdapterTest.java:85]: I would check if list size is same as before.

File: app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/EditViewModelTest.java [hunk 6]
```diff
@@ -1,22 +1,37 @@
...
 
     @Test
     public void whenTagIsSuccessfullyCleared_ClearingIsPropagate() {
         //Given
         EditCompoundRecyclerView editCompoundRecyclerView = mock(EditCompoundRecyclerView.class);
```
Comment (BartSoftware) [inline app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/EditViewModelTest.java]: I see those 5 lines are boilerplate, maybe move them to method?

File: app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/adapter/EditItemAdapterTest.java [hunk 7]
```diff
@@ -1,14 +1,35 @@
...
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(RobolectricTestRunner.class)
 @Config(constants = BuildConfig.class, sdk = 21)
 public class EditItemAdapterTest {
 
-    /*private Tag TAG = new Tag("key", "value", InputType.STRING);
+    private Tag TAG = new Tag("key", "value", TagType.build(InputType.TEXT_STRING));
```
Comment (BartSoftware) [inline app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/adapter/EditItemAdapterTest.java]: final

**Review by BartSoftware — CHANGES_REQUESTED** [review]

**Pull request merged**